hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -510,6 +511,48 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Reconcile global config related configmaps and secrets
+	{
+		configMaps := &corev1.ConfigMapList{}
+		if err := r.List(ctx, configMaps, ctrlclient.MatchingLabels{hostedClusterConfigLabel: hcluster.Name}, ctrlclient.InNamespace(hcluster.Namespace)); err != nil {
```

Comment:
Alternatively to a magic label we could have a `[]localRefs` field in `ClusterConfiguration`. This would be a stronger contract.